packit.yaml

```diff
@@ -26,14 +27,19 @@ jobs:
       - epel-8
       - epel-9
       - fedora-all
-    actions:
+    actions: &build_actions
       post-upstream-clone: make spec
       # HACK: tarball for releases (copr_build, koji, etc.), copying spec's Source0; this
       # really should be the default, see https://github.com/packit/packit-service/issues/1505
       create-archive:
         - sh -exc "curl -L -O https://github.com/osbuild/cockpit-composer/releases/download/${PACKIT_PROJECT_VERSION}/${PACKIT_PROJECT_NAME_VERSION}.tar.gz"
         - sh -exc "ls ${PACKIT_PROJECT_NAME_VERSION}.tar.gz"
 
+  - job: copr_build
+    trigger: release
+    metadata: *build_metadata
+    actions: *build_actions
```

Comment:
Just note that this may be unnecessary. The commit that corresponds to the release would be anyway already built in the repo from `main`, because releases are based on tags (so no new commit). In addition, the release build would be superseded by the next commit on `main` and its build.

So not a problem, it just IMHO adds very little value.

Reply:
Okay thanks for that input, I wasn't a 100% sure so I did it in case.